src/compas/topology/connectivity.py

```diff
@@ -5,9 +5,11 @@
 from compas.utilities import pairwise
 from compas.utilities import geometric_key
 
+from compas.datastructures.network import Network
```

Comment:
topology can't import from datastructures.
maybe this works now, but it will certainly produce errors later

Reply:
also don't import from beyond the second level => `from compas.datastructures import Network`